frontend/src/locales/subscription/en-US.json

```diff
@@ -2,12 +2,11 @@
   "current": "Current plan",
   "instance-count": "Instance count",
   "expires-at": "Expires at",
-  "free-trial": "Your first 14 days are free - no credit card required",
+  "free-trial": "You can start a free trial for 14 days - no credit card required",
```

Comment:
https://github.com/bytebase/bytebase/pull/3217/files#diff-70b9dffa95352e3a42481fe5930e3233e0834d95b1ad37ab66cb3272a43237b1R16 and backend used 7 days but this says 14 days. Shall we make them consistent?

Reply:
That’s quite a question. The 14 days trial means users can log in to the hub and then start the trailing for 14 days. But the 7 days trial only works in the console for the first time.
I prefer not to mention the 14 days trial anymore and only show the 7 days free trial. WDYT?

Reply:
Or let's use 14 days consistently?

Reply:
But no blocking this PR.